ci-operator/config/openshift/release/openshift-release-master__nightly-4.10.yaml

```diff
@@ -166,6 +166,13 @@ tests:
     env:
       TEST_TYPE: upgrade-conformance
     workflow: baremetalds-e2e-upgrade
+- as: e2e-metal-ipi-upgrade-ovn-ipv6
+  cron: 0 0 1 1 *
+  steps:
+    cluster_profile: packet
+    env:
+      TEST_TYPE: upgrade
```

Comment:
Shouldn't be `upgrade-conformance` as for the others?

Reply:
If this basic one passes, I'll try with `upgrade-conformance`. Because I'm not sure conformance also handles skipping disconnected tests.

Reply:
As long as the setup is specifying that's [ipv6](https://github.com/openshift/release/blob/3817e2ace170408cf8cc05c83ce81d979ff40f81/ci-operator/step-registry/baremetalds/e2e/test/baremetalds-e2e-test-commands.sh#L30-L39), I think it should be fine for our step, since the conformance upgrade runs the parallel tests after the upgrade. But agree, let's see the simple upgrade output